src/logic/tokens/store/actions/fetchTokens.ts

```diff
@@ -100,11 +100,13 @@ export const fetchTokens = () => async (
       data: { results: tokenList },
     } = await fetchTokenList()
 
-    if (currentSavedTokens && currentSavedTokens.size === tokenList.length) {
+    const erc20Tokens = tokenList.filter((token) => token.type.toLowerCase() === 'erc20')
```

Comment:
Should we move `erc20` string to a constant variable?

Reply:
I don't think it's mandatory now. Probably when we work on #1408 we'll address this in a most consistent way.